plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -242,97 +237,305 @@ public void testCharVarcharComparison()
     @Test
     public void testPushdownUuidPartitionKeyPredicate()
     {
-        assertThat(query(format("SELECT col_text FROM %s.%s WHERE col_uuid = UUID '00000000-0000-0000-0000-000000000001'", KEYSPACE, TABLE_PUSHDOWN_UUID_PARTITION_KEY_PREDICATE)))
-                .matches("VALUES CAST('Trino' AS varchar)");
+        try (TestCassandraTable testCassandraTable = new TestCassandraTable(
+                session::execute,
+                server,
+                KEYSPACE,
+                "table_pushdown_partition_key_predicate",
+                ImmutableList.of(partitionColumn("col_uuid", "uuid"), generalColumn("col_text", "text")),
+                ImmutableList.of("00000000-0000-0000-0000-000000000001, 'Trino'"))) {
+            assertThat(query(format("SELECT col_text FROM %s WHERE col_uuid = UUID '00000000-0000-0000-0000-000000000001'", testCassandraTable.getTableNameInKeyspace())))
+                    .matches("VALUES CAST('Trino' AS varchar)");
+        }
     }
 
     @Test
     public void testPushdownAllTypesPartitionKeyPredicate()
     {
         // TODO partition key predicate pushdown for decimal types does not work https://github.com/trinodb/trino/issues/10927
-        String sql = "SELECT *" +
-                " FROM " + TABLE_PUSHDOWN_ALL_TYPES_PARTITION_KEY_PREDICATE +
-                " WHERE key = 'key 7'" +
-                " AND typeuuid = UUID '00000000-0000-0000-0000-000000000007'" +
-                " AND typetinyint = 7" +
-                " AND typesmallint = 7" +
-                " AND typeinteger = 7" +
-                " AND typelong = 1007" +
-                " AND typedate = DATE '1970-01-01'" +
-                " AND typetimestamp = TIMESTAMP '1970-01-01 03:04:05Z'" +
-                " AND typeansi = 'ansi 7'" +
-                " AND typeboolean = false" +
-                " AND typedouble = 16384.0" +
-                " AND typefloat = REAL '2097152.0'" +
-                " AND typeinet = '127.0.0.1'" +
-                " AND typevarchar = 'varchar 7'" +
-                " AND typetimeuuid = UUID 'd2177dd0-eaa2-11de-a572-001b779c76e7'" +
-                "";
-        MaterializedResult result = execute(sql);
+        try (TestCassandraTable testCassandraTable = new TestCassandraTable(
+                session::execute,
+                server,
+                KEYSPACE,
```

Comment:
I would recommend creating a helper method in this class for building `TestCassandraTable` object because these 3 constants are always same and reduce readability. 

```java
    private TestCassandraTable testTable(String namePrefix, List<TestCassandraTable.ColumnDefinition> columnDefinitions, List<String> rowsToInsert)
    {
        return new TestCassandraTable(session::execute, server, KEYSPACE, namePrefix, columnDefinitions, rowsToInsert);
    }
```